inorbit_edge/robot.py

```diff
@@ -26,6 +29,9 @@
 MQTT_POSE_TOPIC = "ros/loc/data2"
 MQTT_TOPIC_CUSTOM_DATA = "custom"
 MQTT_TOPIC_ODOMETRY = "ros/odometry/data"
+MQTT_TOPIC_PATH = "ros/loc/path"
+
+ROBOT_PATH_POINTS_LIMIT = 1000
```

Comment:
Where does this limit come from ? are we doing the same on the agent ?

Reply:
I guess we can implement some kind of downsampling, or leave that responsibility to the SDK user.

Reply:
I just wanted to understand this better, the hard limit seems ok at least for now!